src/styles/blocks/container.scss

```diff
@@ -0,0 +1,24 @@
+.container {
+  margin: 0 auto;
+  width: $container-width;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding: 50px 0;
+
+  &__card {
+    margin-right: 48px;
+  }
+
+  & > :nth-child(4n) {
+    margin-right: $reset;
+  }
+
+  & > :nth-last-child(-n+4) {
+    margin-top: 46px;
+  }
+
+  @media screen and (min-width: 1024px) and (max-width: 1200px) {
```

Comment:
In the media query, the width is being set to 1024px. This may cause horizontal scrolling on devices with a viewport width of less than 1024px. Consider using a grid